utils/ui.js

```diff
@@ -29,6 +29,28 @@ function isElementInView(el, opt_offset) {
 }
 
 
+/**
+ * Creates a style element and inserts it into the head of the document.
+ * @param {Array.Array} A list of [<string>, <string>] elements where the first
+ *     element is the selector to create and the second element is the
+ *     corresponding style. Example: [['.foo', 'display: none']]
+ * @return {Element} Created style element.
+ */
+function createStyle(selectorsAndValues) {
+  var content = '';
```

Comment:
String concatenation is expensive, use string building using an array and join at the end.